README.md

```diff
@@ -36,8 +36,8 @@ BOOL crashedOnPriorLaunch = ...
 id<UBApplicationStartupReasonReporterPriorRunInfoProtocol> runInfo = ...
 
 // Initialize the startup reason reporter
-UBApplicationStartupReasonReporter *applicationStartupReasonReporter = [[UBApplicationStartupReasonReporter alloc] initWithNotificationCenter:[NSNotificationCenter defaultCenter]
-        previousRunDidCrash:self.crashReporter.crashDetected
```

Comment:
This makes it less obvious that you'd need to pull this information from your crash reporting tool.

Reply:
This change matches the swift. It also matches the fact that you've called the local variable `crashedOnPriorLaunch` three lines above.

Reply:
Sorry, did a too hurried review. Right you are.